docs/overview/getting-started/intro.md

```diff
@@ -2,69 +2,53 @@
 slug: /intro
 ---
 
-# Overview
+# Introduction
 
-## Introduction
+Avalanche is an open-source platform for building decentralized applications (Dapps) in one
+interoperable, highly scalable ecosystem. Avalanche is the first ecosystem designed to accommodate
+the scale of global finance, with near-instant transaction finality.
 
-[Avalanche](https://avax.network) is an open-source platform for launching [decentralized
-applications](https://support.avalabs.org/en/articles/4587146-what-is-a-decentralized-application-dapp)
-and enterprise [blockchain](http://support.avalabs.org/en/articles/4064677-what-is-a-blockchain)
-deployments in one interoperable, highly scalable ecosystem. Avalanche is the first decentralized
-smart contracts platform built for the scale of global finance, with near-instant transaction
-finality. Ethereum developers can quickly build on Avalanche as Solidity works out-of-the-box.
+Avalanche empowers developers to join a rich ecosystem of blockchains known as Subnets. Subnets
+are isolated clusters of the Avalanche network that allow any Dapp to run on its own dedicated set
```

Comment:
```suggestion
are isolated clusters of the Avalanche network that allow any dApp to run on its own dedicated set
```

Reply:
Yeah i originally had it written this way but was concerned since I wanted the intro page to be easily readable by a non-technical audience. I was probably too worried though, I think the latter is more accurate and is still understandable. I'll update to this wording.